src/pages/home/report/ReportActionItemMessage.js

```diff
@@ -2,18 +2,30 @@ import React from 'react';
 import {View} from 'react-native';
 import PropTypes from 'prop-types';
 import _ from 'underscore';
+import {withOnyx} from 'react-native-onyx';
+import ONYXKEYS from '../../../ONYXKEYS';
 import styles from '../../../styles/styles';
 import ReportActionItemFragment from './ReportActionItemFragment';
 import ReportActionPropTypes from './ReportActionPropTypes';
 
 const propTypes = {
     // The report action
     action: PropTypes.shape(ReportActionPropTypes).isRequired,
+
+    // Information about the network
+    network: PropTypes.shape({
+        // Is the network currently offline or not
+        isOffline: PropTypes.bool,
+    }),
+};
+
+const defaultProps = {
+    network: {isOffline: false},
 };
 
-const ReportActionItemMessage = ({action}) => {
+const ReportActionItemMessage = ({action, network}) => {
     // reportActionID is only present when the action is saved onto server.
-    const isUnsent = action.loading && !action.reportActionID;
+    const isUnsent = network.isOffline && action.loading && !action.reportActionID;
```

Comment:
I'm not sure we need to check both `action.loading`  and `action.reportActionID`? What does having a `reportActionID` even mean?

Reply:
Or asked another way are there comments that are "not loading" but "don't have a reportActionID"... it's unclear how we end up in that situation to me.

Reply:
I used this to make a difference from the attachment files. As they are also in loading status until they are uploaded. `action.reportActionID` is only present when comment is uploaded to backend.

Reply:
Yeah, it makes sense. we don't actually need it.

Reply:
>they are also in loading status until they are uploaded

Ok, yeah I believe there's no difference there. It should be the same whether attachment or not.